### DISCUSSION TITLE
Infinite ammo and boost for Vehicles & Bypassing Casino Region

### DISCUSSION THREAD

**Discussion by EbneSinam** [YimMenu/YimMenu#1012] [Ideas]
Hi, Please Add
1.Infinite ammo and automatic charge boost for Vehicles
2.Bypassing Casino Region restrictions
Thankful.

**Comment by Aure7138** [comment]
To implement Bypassing Casino Region, can simply hook NETWORK_CASINO_CAN_BET and return TRUE to achieve
![imgae](https://i.imgur.com/g1xRLv1.png)
![image](https://i.imgur.com/XyPXakQ.png)

**Reply by EbneSinam** [reply]
How can I find and change this option? Sorry, I'm new

**Reply by Primexz** [reply]
> How can I find and change this option? Sorry, I'm new

i will push a pr for this feature soon

**Reply by EbneSinam** [reply]
> i will push a pr for this feature soon

Thank you

**Reply by EbneSinam** [reply]
Excuse me, how can I activate this option? In which part is it? I did not find it.
The casino is still limited for me. I'm from Iran.

**Reply by EbneSinam** [reply]
> Excuse me, how can I activate this option? In which part is it? I did not find it. The casino is still limited for me. I'm from Iran.

Thank you, it was fixed in this update

**Comment by EbneSinam** [comment]
I found the option of automatic boost charging. (Fun Features => Boost Behavior => Instant Refill)

But I didn't find an option for infinite ammo. For example, the Oppressor Mk II is limited to 20 Missiles and will not fire after 20 Missiles
